irohad/ametsuchi/mutable_storage.hpp

```diff
@@ -40,11 +39,10 @@ namespace iroha {
     class MutableStorage {
      public:
       /**
-       * Predicate type checking type T
+       * Predicate type checking block
        */
-      template <typename T>
       using MutableStoragePredicateType =
```

Comment:
Maybe then rename it in order to clarify that this predicate is now about block only?

Reply:
Can you suggest something? I think this name is okay now. It did not reflect template in the type before, so now it is also not needed.